ocaml/xenopsd/scripts/qemu-wrapper

```diff
@@ -310,7 +309,7 @@ def main(argv):
 
     clipboardd = '/opt/xensource/libexec/xs-clipboardd'
     subprocess.call([clipboardd, "-d", str(domid), "-s", str(s2.fileno())],
-                    preexec_fn=close_fds)
+                    pass_fds=[s2.fileno()])
```

Comment:
The code could probably do with some refactoring after this change. `close_fds` is short and only used once so it could probably be inlined. It also weirdly relies on global state rather than taking a parameter.

This refactoring can be done later.